### PR TITLE
Showcase : To do list App added

### LINKED ISSUE #657: [Feature Request]:  To add to do list app in showcase
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

i would like to add to do list in showcase 

### Use Case

addition of one more showcase will me helpful

### Benefits

_No response_

### Add ScreenShots

image

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by dhairyagothi**
## Related Issue

Fix #657 

## Description

Added to do list app in showcase section

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

image

## Checklist
- [X] I have performed a self-review of my code.
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: src/data/users.tsx [hunk 1]
```diff
@@ -263,6 +263,15 @@ const Users: User[] = [
     source: "https://github.com/ParnaRoyChowdhury777/Age-Calculator",
     tags: ["opensource", "product", "react", "css", "javascript"],
   },
+  {
+    title: "To Do List App",
+    description:
+      "A simple To-Do List App using HTML CSS JS . Add your tasks and manage them easily. ",
+    preview: require("./showcase/todolist.png"),
+    website: "https://50-days-50-web-project.vercel.app/TO_DO_LIST/todolist.html",
```
Comment (Ajay-Dhangar) [inline src/data/users.tsx:271]: @dhairyagothi can you update you web page UI, layout  for best practices and good page for motivate and good for users to work build like this and for best UI/UX

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Please update your project design for the valuable page or resources, as the current design is not suitable for our showcase.

**Comment by dhairyagothi** [comment]
please check the PR now and please tell what extra changes you want in to do list app ? i will make changes in base repo of to do list app

**Review by Ajay-Dhangar — APPROVED** [review]
okay

**Review by Ajay-Dhangar — APPROVED** [review]
okay, based on image is good so, now update like this

**Comment by dhairyagothi** [comment]
> okay, based on image is good so, now update like this

okay for sure i will do please merge the PR

**Pull request merged**